src/app/leverage/components/leverage-widget/index.tsx

```diff
@@ -99,6 +104,7 @@ export function LeverageWidget(props: LeverageWidgetProps) {
       <BuySellSelector isMinting={isMinting} onClick={toggleIsMinting} />
       <LeverageSelector
         selectedTye={leverageType}
+        supportedTypes={supportedLeverageTypes}
```

Comment:
Mainnet only supports `2x`. So this makes sure the selector has the according buttons.